libraries/AP_NavEKF2/AP_NavEKF2_Measurements.cpp

```diff
@@ -180,6 +180,54 @@ void NavEKF2_core::writeOptFlowMeas(uint8_t &rawFlowQuality, Vector2f &rawFlowRa
     }
 }
 
+// reset variances for mag body state
+void NavEKF2_core::resetMagBodyVariances(void)
+{
+    zeroCols(P,19,21);
+    zeroRows(P,19,21);
+    P[19][19] = sq(frontend->_magNoise);
+    P[20][20] = P[19][19];
+    P[21][21] = P[19][19];
+}
+
```

Comment:
What is significant about 19, 20, and 21?  Can these magic numbers be #define, const, or constexpr?

Reply:
Or at least put a comment explaining 19 - 21?